app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :body, presence: true
+  validates :body, length: { minimum: 3 }
```

Comment:
You can write like this :
`validates :body, presence: true, length: { minimum: 3 }`

Reply:
Changes have been applied